packages/plugins/persisted-operations/src/index.ts

```diff
@@ -1,12 +1,23 @@
-import { DocumentNode, GraphQLErrorOptions } from 'graphql'
+import { ASTNode, DocumentNode, GraphQLErrorExtensions, Source } from 'graphql'
 import {
   createGraphQLError,
   GraphQLParams,
   Plugin,
   PromiseOrValue,
   type OnParamsEventPayload,
+  Maybe,
 } from 'graphql-yoga'
 
+// GraphQLErrorOptions interface does not exist in graphql-js v15
+export interface GraphQLErrorOptions {
```

Comment:
We could actually export this from utils, and reuse it here;
https://github.com/ardatan/graphql-tools/blob/master/packages/utils/src/errors.ts#L17

Reply:
It doesn't export the GraphQLErrorOptions though, we'd need to fix and bump utils first.